### ISSUE TITLE
CLI - need a review of defaulted option values

### ISSUE THREAD

**Issue by sallyom** [backube/scribe#146]
There are many API fields that have defaulted values and these are included in the cli options. There are a few that have been defaulted in CLI options:
```
destination pvc capacity: "2Gi"
source pvc capacity: "2Gi"
source-cron-spec: "*/3 * * * *"
```
Should others be defaulted? Should these not have defaults?

**Comment by sallyom** [comment]
> can't we pull the source pvc capacity dynamically and then default the dest capacity based on the source?

yes, i'll make that change/on list of todos  - then we need to decide if source PVC should be defaulted or should user be required to always provide that.

**Comment by JohnStrunk** [comment]
Agree w/ Scott. Let's look at the source PVC for capacities unless otherwise specified.
- I might go w/ a 5min replication interval... just more of a round number :man_shrugging: 
- I wish there were something we could do to default the copyMethod, but I think it's too dependent on the driver. Users should favor "Clone" on source and "Snapshot" on dest, but when EBS doesn't support clone, that's a hard argument to make.
- StorageClass and VSC should both default to nil so that the cluster defaults take over
- And I'd probably stick w/ ClusterIP for the service type on the dest since that's less scary.

**Comment by sallyom** [comment]
Hm, the problem with defaulting to source-pvc capacity, that I see now as I'm looking at this closer, is that it's very much ok to create a replicationDestination b4 a replicationSource, ie, it's not required that a replicationSource exists. We _could_ introduce a destination flag `source-name` or something, that will be the name of a replicationSource and (after a slight refactor of the common options that I knew would probably cause issues) add a flag `--source-namespace` for the destination (atm there's only source-ns for new-source and dest-ns for new-dest) and/or accept a `--source-pvc-name` and cli can either inspect the source-pvc or replicationSource to get the capacity value, but at that point might-as-well pass a `--dest-capacity` :shrug:

**Comment by JohnStrunk** [comment]
Perhaps we're treading on where #148 is going... the current new-source, new-destination treat each in isolation, making the defaulting hard. Maybe we skip most defaulting for these "low level" operations and focus on the higher-level stuff?

How much (user) effort can we save if we have them express: "set up async replication from c1/ns1/pvc1 -> c2/ns2/pvc2"? Even before we get to a specific usage of replication (e.g., data migration), I think we can provide a good bit of savings. Having them express what they want to accomplish gives us the context we need to help them.

**Comment by sallyom** [comment]
yes, agreed, maybe close this issue and move discussions over to that (bigger) issue?

**Comment by sallyom** [comment]
I'll update the replication interval to 5 min, agreed, round number.
